### DISCUSSION TITLE
Datenverzeichnis extern mounten NFS

### DISCUSSION THREAD

**Discussion by electric-m** [camueller/SmartApplianceEnabler#179] [Installation und Betrieb]
Ich habe SAE unter Docker auf dem Raspi laufen.
Gerne wollte ich das Datenverzeichnis aus dem Container in eine NFS Freigabe mounten.
Mit vielen anderen Conatinern mache ich das bereits erfolgreich.

Nach meinem Verständnis müsste das entweder /opt/sae oder /opt/sae/data sein?

Bei /opt/sae wird der Container zwar gestartet, das Verzeichnis bleibt aber leer. Liegt das daran,. dass ich noch keine Geräte angelegt habe?

/opt/sae/data funktioniert nicht.

**Comment by electric-m** [comment]
UPDATE:

Nach Anlage eines ersten Gerätes werden die XML im Verzichnis /opt/sae/data angezeigt.

**Comment by camueller** [comment]
`/opt/sae/data` ist ein Docker-Volume, auf dem sich die beiden XML-Files befinden. Das Executable (*.war) und die Logger-Konfiguration befinden sich direkt in `/opt/sae`, da sie zum Image gehören.

Wozu willst Du das als NFS-Freigabe machen? Du kannst doch auch mit `docker cp ...` Dateien zwischen dem Volume und dem Host hin- und her kopieren.

**Reply by electric-m** [reply]
So habe ich immer im Finder direkt Zugriff drauf.
Und falls der Container oder der Pi mal abschmiert, liegen die Dateien auf jeden Fall auf dem NFS Share.
Funktioniert ja jetzt.
